x-pack/test/spaces_api_integration/common/fixtures/es_archiver/saved_objects/spaces/data.json

```diff
@@ -49,3 +49,56 @@
     }
   }
 }
+
+{
+  "type": "doc",
+  "value": {
+    "index": ".kibana",
+    "type": "doc",
+    "id": "space_2:dashboard:my_dashboard",
+    "source": {
+      "type": "dashboard",
+      "updated_at": "2017-09-21T18:49:16.270Z",
+      "namespace": "space_2",
+      "dashboard": {
+        "description": "Space 2",
+        "title": "This is the second test space"
+      }
+    }
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "index": ".kibana",
+    "type": "doc",
+    "id": "space_1:dashboard:my_dashboard",
+    "source": {
+      "type": "dashboard",
+      "updated_at": "2017-09-21T18:49:16.270Z",
+      "namespace": "space_1",
+      "dashboard": {
+        "description": "Space 1",
+        "title": "This is the second test space"
+      }
+    }
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "index": ".kibana",
+    "type": "doc",
+    "id": "dashboard:my_dashboard",
+    "source": {
+      "type": "dashboard",
+      "updated_at": "2017-09-21T18:49:16.270Z",
+      "dashboard": {
+        "description": "Space 1",
```

Comment:
nit: the description/title for the test data make it appear that we're in Space 1 but they exist in the default space.